components/header.tsx

```diff
@@ -0,0 +1,37 @@
+import { navOptions, title } from '@constants/header';
+import Link from 'next/link';
+import React from 'react';
```

Comment:
No need for the react import. Keep library imports at top. Add line gap between library and internal imports